File: FABPortfolioApp.SPA/src/app/_models/portfolio.ts
import { PortfolioFile } from './portfolioFile';

export interface Portfolio {
  id: number;
  project: string;
  description: string;
  url: string;
  from: string;
  to: string;
  company: string;
  location: string;
  portfolioFiles: PortfolioFile;
}

// portfolioPictureId: number;
// portfolioPictures: PortfolioPicture;

File: FABPortfolioApp.API/Models/Portfolio.cs
using System;
using System.Collections.Generic;

namespace FABPortfolioApp.API.Models
{
    public class Portfolio
    {
        public int Id { get; set; }
        public string Project { get; set; }
        public string Company { get; set; }
        public string Location { get; set; }
        public string From { get; set; }
        public string To { get; set; }
        public string Url { get; set; }
        public ICollection<PortfolioFile> PortfolioFiles { get; set; }
    }
}

